src/tools/cephfs/cephfs-shell

```diff
@@ -1333,7 +1334,10 @@ if __name__ == '__main__':
     if args.config:
         config_file = args.config
     if args.batch:
-        args.commands = ['load ' + args.batch, ',quit']
+        if cmd2_version <= "0.9.13":
```

Comment:
Careful here: you cannot use string comparisons for "semantic versioning". e.g., if `cmd2_version` is `0.19.13`, it will wrongly be `<= 0.9.13`.

See also: https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python/21065570

Reply:
Nice catch. :D  

I've made the change - https://github.com/ceph/ceph/pull/30585/files#diff-9226892dd49173c4687a9953904ccaa8R1338. Thanks!